apps/configurator/lib/configurator_web/controllers/skill_controller.ex

```diff
@@ -0,0 +1,69 @@
+defmodule ConfiguratorWeb.SkillController do
+  use ConfiguratorWeb, :controller
+
+  alias GameBackend.Units.Characters
+  alias GameBackend.Units.Skills
+  alias GameBackend.Units.Skills.Mechanic
+  alias GameBackend.Units.Skills.Skill
+  alias GameBackend.Utils
+
+  def index(conn, _params) do
+    config_skills = Skills.list_config_skills()
```

Comment:
We're filtering the champions character by game_id, can we do the same for the skills?